test/e2e/mock-3box/server.js

```diff
@@ -20,7 +19,7 @@ const requestHandler = (request, response) => {
       response.end('ok')
     })
   } else if (request.method === 'GET') {
-    const { key } = url.parse(request.url, true).query
+    const key = (new URL(request.url, 'https://example.org/')).searchParams.get('key')
```

Comment:
Seeing `searchParams` used reminded me of this: https://twitter.com/domenic/status/1257377082704900096

I was curious, so I checked to see if there was any behaviour change here between `url` and `URL`. It looks as though there is not. `url` parses the query string in the same weird way that URL does, at least with the examples listed in that PR description anyway. I didn't try the linked exhaustive list. Of course it doesn't matter here for this mock server anyway.